examples/basic/events/AddedItemToCart/index.md

```diff
@@ -12,6 +12,10 @@ domains:
 owners:
     - dboyne
     - mSmith
+extLinks: 
```

Comment:
@thim81 what do you think of having `externalLinks`, might be slightly more descriptive , `extLinks` I think might require some lookup or understanding? What do you think?

Reply:
Fully right, I'll update all the changes to use externalLinks